catalog-info.yaml

```
@@ -150,6 +150,42 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-macos-integration
+  description: Elastic Agent macos integration test pipeline
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elastic-agent-macos-integration
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: platform-ingest
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elastic-agent-macos-integration
+      description: Elastic Agent macos integration test pipeline
```

I've seen some references for `Elastic Agent extended testing using buildkite agents` in `L137`, so is `extended testing`, what we aim to put in this description?

```suggestion
      description: Elastic Agent extended testing using Orka agents
```